deploy/codex/README.md

```diff
@@ -45,7 +45,7 @@ Precision nuke example:
 
     docker-compose stop houston && docker-compose rm -f houston && docker volume rm codex_houston-var
 
-Docker is conservative about cleaning up unused objects. This can cause Docker to run out of disk space or 
+Docker is conservative about cleaning up unused objects. This can cause Docker to run out of disk space or
```

Comment:
Is this file intentional or a rebase error?

Reply:
this was the linting tools doing.... whatever they are doing with some sort of dangly whitespace.